bowling/inc/fileReader.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <experimental/filesystem>
+#include <memory>
+#include <string>
+
+#include "lane.hpp"
+
+class FileReader {
```

Comment:
My first thought was that `FileReader` class is for reading a single file, but it turns that its task is to read multiple files from a folder. Maybe `FilesReader` (plural) would be a better name or `DataReader` or `FolderReader`? We can discuss it with the rest of our group on Discord.